<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvDcConnectedNodeWidget</class>
 <widget class="QWidget" name="C_SyvDcConnectedNodeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pc_LabelIcon">
       <property name="text">
        <string>Icon</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="stw_opensyde_gui_elements::C_OgeLabGroupSubItem" name="pc_LabelNodeId">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="stw_opensyde_gui_elements::C_OgeLabPropertiesSubHeading" name="pc_LabelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="stw_opensyde_gui_elements::C_OgeLabGroupSubItem" name="pc_LabelDeviceType">
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabPropertiesSubHeading</class>
   <extends>QLabel</extends>
   <header>C_OgeLabPropertiesSubHeading.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabGroupSubItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupSubItem.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
